api-spec/payment-requests-api.yaml

```
@@ -696,4 +703,11 @@ components:
         description:
           description: Payment notice description
           type: string
-          maxLength: 140 # as paymentDescription for paForNode
\ No newline at end of file
+          maxLength: 140 # as paymentDescription for paForNode
+    ClientId:
+      type: string
+      description: Enumerations of client ids set by APIM
+      enum:
+        - WISP_REDIRECT
+        - CHECKOUT
+        - CHECKOUT_CART
```

```suggestion
    ClientId:
      type: string
      description: Enumerations of client ids set by APIM
      enum:
        - WISP_REDIRECT
        - CHECKOUT_CART
```
just thinking, as for now the only valid values for POST carts header are WISP_REDIRECT and CHECKOUT_CART as per apim policies so I would remove CHECKOUT. What do you think?